File: src/main/java/pe/com/test_qsystem/storage/impl/S3FileStorage.java
package pe.com.test_qsystem.storage.impl;

import pe.com.test_qsystem.storage.FileStorage;

public class S3FileStorage implements FileStorage {

    @Override
    public void saveFile(String path) {
        System.out.println("save file on aws s3.");
    }
    
}

File: src/main/kotlin/pe/com/qsystem/test_qsystem/storage/impl/S3FileStorage.kt
package pe.com.qsystem.test_qsystem.storage.impl

import org.slf4j.LoggerFactory
import pe.com.qsystem.test_qsystem.storage.FileStorage

class S3FileStorage : FileStorage {
    
    private val log = LoggerFactory.getLogger(javaClass)
    
    override fun saveFile(path: String) {
        log.info("Save file on aws s3.")
    }
}